src/components/CancelBlock.tsx

```diff
@@ -132,16 +113,25 @@ export const CancelBlock: FunctionComponent = () => {
           </OrangeButton>
         </div>
       </div>
-
-      <div className="w-100 h-20 max-w-screen-lg w-full mt-6 mx-auto ">
-        <p className={"my-2 text-sm text-gray-700"}>Status </p>
-        <LoggerStyle
-          className={"w-full h-16 bg-gray-100 p-2 text-sm resize-none overflow-scroll"}
-          data-testid="cancel-log"
+      <Logger log={logs} className="px-4" />
```

Comment:
there weren't enough changes ? :)

What does this do

Reply:
instead of having LoggerStyle created in each individual block. I decided to segment them out into a logger component. easier for me to update all the logger style at one go